src/components/ContextBar.js

```diff
@@ -0,0 +1,66 @@
+import { Fragment } from 'react'
+import { div, h } from 'react-hyperscript-helpers'
+import { Clickable, comingSoon, makeMenuIcon, MenuButton } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import PopupTrigger from 'src/components/PopupTrigger'
+import colors, { terraSpecial } from 'src/libs/colors'
+
+
+const contextBarStyles = {
+  contextBarContainer: {
+    display: 'flex', flexWrap: 'wrap'
+  },
+  contextBarButton: {
+    display: 'flex', //, justifyContent: 'center', alignItems: 'center',
+    borderBottom: `1px solid ${colors.accent()}`,
+    padding: '1rem 1rem 1rem 1rem',
```

Comment:
Since the padding is all 1rem you can actually just do `padding: 1rem` and it will apply it to all the directions automatically